src/amo/components/AddonReviewList/index.js

```diff
@@ -247,7 +248,12 @@ export class AddonReviewListBase extends React.Component<Props> {
       ) : null;
 
     return (
-      <div className="AddonReviewList">
+      <div
+        className={makeClassName(
+          'AddonReviewList',
+          `AddonReviewList--${(addon && addon.type) || ''}`,
```

Comment:
With `makeClassName`, you can have conditions with objects (the keys being the class names and the values tghe conditions):

```js
makeClassName('AddonReviewList, {
  'AddonReviewList--${addon.type}: addon && addon.type, 
})
```

Reply:
I actually had something similar but slightly diff b/c it's a dynamic class name:
```javascript
   <div
        className={makeClassName('AddonReviewList', {
          [`AddonReviewList--${addon.type}`]: addon && addon.type,
        })}
      >
```
but flow complains about this :/

is it better to just hard code the class name..just for static theme type?

Reply:
What's the complaint?

Reply:
ah I think I know. Then:

```
  ...
  // Flow requires an extra check in the class name.
  [`AddonReviewList--${addon && addon.type}`]: addon && addon.type,
```

Reply:
```bash
Cannot get addon.type because property type is missing in null [1].

 [1]  40│   addon: AddonType | null,
        :
     250│     return (
     251│       <div
     252│         className={makeClassName('AddonReviewList', {
     253│           [`AddonReviewList--${addon.type}`]: addon && addon.type,
     254│         })}
     255│       >
     256│         {addon && (

```

Reply:
ok let me check that. I thought I tried that too :/ one sec..

Reply:
sorry this is the top part of the flow info:
`Cannot coerce addon && addon.type to string because null [1] should not be coerced.`

Reply:
full response:
```bash
Cannot coerce addon && addon.type to string because null [1] should not be coerced.

 [1]  40│   addon: AddonType | null,
        :
     250│     return (
     251│       <div
     252│         className={makeClassName('AddonReviewList', {
     253│           [`AddonReviewList--${addon && addon.type}`]: addon && addon.type,
     254│         })}
     255│       >
     256│         {addon && (
```

Reply:
Well, then let's write maybe:

```
makeClassName('foo', addon && addon.type ? addon.type : null)
```